packages/babel-core/src/index.js

```diff
@@ -1,39 +1,42 @@
-export File from "./transformation/file";
```

Comment:
Can we continue to export `File`? I need it in a number of places to create nice error messages

Reply:
@loganfsmyth